tests/conftest.py

```diff
@@ -123,6 +123,10 @@ def log_ocs_version(cluster):
      * ocs_version file in cluster path directory (for copy pasting into bug
        reports)
     """
+    teardown = config.RUN['cli_params']['teardown']
+    if teardown:
```

Comment:
It is not typical usage, but what if the `run-ci` will be executed with both `--deploy` and `--teardown` at once? Then it will not report the versions even though it might be expected.
What about something like:
```python
if teardown and not deploy:
    ...
```